src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -753,7 +753,8 @@ protected DynamoModel(IStartConfiguration config)
                 if (migrator != null)
                 {
                     var isFirstRun = PreferenceSettings.IsFirstRun;
-                    PreferenceSettings.Instance = migrator.PreferenceSettings;
+                    PreferenceSettings = migrator.PreferenceSettings;
```

Comment:
Shouldn't `PreferenceSettings.Instance` be updated here as well?

Reply:
I thought originally and had it that way.  @QilongTang pointed out that it shouldn't need an update.  

We have:
PreferenceSettings = foo;
Instance = PreferenceSettings
PreferenceSettings = otherFoo;

I don't think you need to reset the Instance right?

Reply:
```
        string PreferenceSettings = "val1";
        string Instance = PreferenceSettings;
        PreferenceSettings = "val2";
        
        Console.WriteLine (PreferenceSettings);
        Console.WriteLine (Instance);
       -----------
        val2
        val1
        ----------
```

Reply:
Ok let me fix that :-)